tools/azure-sdk-tools/devtools_testutils/resource_testcase.py

```diff
@@ -65,22 +69,29 @@ def create_resource(self, name, **kwargs):
             parameters = {"location": self.location}
             if self.delete_after_tag_timedelta:
                 expiry = datetime.datetime.utcnow() + self.delete_after_tag_timedelta
-                parameters["tags"] = {
-                    "DeleteAfter": expiry.replace(microsecond=0).isoformat()
-                }
+                parameters["tags"] = {"DeleteAfter": expiry.replace(microsecond=0).isoformat()}
```

Comment:
This looks like it ends up being optional which might explain some of the RG's that don't have a DeleteAfter tag. Can we instead ensure we have a default time delta so that this DeleteAfter always gets added.

Reply:
Also it looks like above the value is set to 1 day we might want to lower that to 8 hours.

Reply:
It's not optional, the default is 1 day, I will lower it to 8 hours.

Reply:
Why is there a check for the value? If it isn't optional we should always just add it.

Reply:
Good point, if a user is setting `delete_after_tag_timedelta=None` this will be caught and we can investigate that situation. I'll make the fix.